packages/automator/index.tsx

```diff
@@ -222,13 +268,14 @@ const batchProcess = async (
     // try to render the diagram
     try {
       // Warning: will face id conflicts if parallelism used
-      const res = await singleProcess(
+      let res = await singleProcess(
         subURI,
         styURI,
         dslURI,
         folders,
         `${out}/${name}-${id}${folders ? "" : ".svg"}`,
         prefix,
+        staged, // make something else to fix singleProcess?
```

Comment:
Is there something wrong with `singleProcess`?

Reply:
woops I'm not sure why I made a comment there 😅 `singleprocess` is fine (i think that was when I was doing some testing)